wdtk-client/src/main/java/org/wikidata/wdtk/client/Client.java

```diff
@@ -151,6 +155,12 @@ public void performActions() {
 		openActions();
 		this.dumpProcessingController.processDump(dumpFile);
 		closeActions();
+		try {
+			createReport();
+		} catch (IOException e) {
+			logger.error("Could not print report file!");
```

Comment:
In the prepareSites method the exception message is also missing. Should I change this there too?